transaction-status/src/extract_memos.rs

```diff
@@ -9,13 +10,18 @@ use {
 // A helper function to convert spl_memo::v1::id() as spl_sdk::pubkey::Pubkey to
 // solana_sdk::pubkey::Pubkey
 pub fn spl_memo_id_v1() -> Pubkey {
-    Pubkey::new_from_array(spl_memo::v1::id().to_bytes())
+    *MEMO_PROGRAM_ID_V1
 }
 
 // A helper function to convert spl_memo::id() as spl_sdk::pubkey::Pubkey to
 // solana_sdk::pubkey::Pubkey
 pub fn spl_memo_id_v3() -> Pubkey {
-    Pubkey::new_from_array(spl_memo::id().to_bytes())
+    *MEMO_PROGRAM_ID_V3
+}
+
+lazy_static! {
+    static ref MEMO_PROGRAM_ID_V1: Pubkey = Pubkey::new_from_array(spl_memo::v1::id().to_bytes());
+    static ref MEMO_PROGRAM_ID_V3: Pubkey = Pubkey::new_from_array(spl_memo::id().to_bytes());
```

Comment:
totally aside: it'd be much easier if all of the `pubkey!` / `declare_id!` and friends just created `const`s everywhere to avoid these.

Reply:
Agreed